### DISCUSSION TITLE
annotation z-index does not work

### DISCUSSION THREAD

**Discussion by Georodin** [google/model-viewer#3539] [Q&A]
Hey guys and gals,
how can I change the z-index effectively of the annotations? 
I want them to come to the front after an on click. Is this possible?
I tried changing the z-index of the `<div class="HotspotAnnotation">` but to no avail.

![grafik](https://user-images.githubusercontent.com/45157241/174607162-ffb86397-d87c-4873-8bb5-d30f059bc739.png)

**Comment by Georodin** [comment]
Duplicate of https://github.com/google/model-viewer/discussions/3029

**Reply by elalish** [reply]
I might recommend hiding the others instead (at least the big text boxes) so you only show one at a time on click.
